com.unity.render-pipelines.universal/Shaders/Nature/SpeedTree7BillboardPasses.hlsl

```diff
@@ -31,7 +31,12 @@ void InitializeData(inout SpeedTreeVertexInput input, out half2 outUV, out half
 
 #ifdef ENABLE_WIND
     if (_WindQuality * _WindEnabled > 0)
+    {
+        // Disabling "pow(f,e) will not work for negative f"; warnings.
+        #pragma warning (disable : 3571)
         billboardPos = GlobalWind(billboardPos, worldPos, true, _ST_WindVector.xyz, input.texcoord1.w);
```

Comment:
Minor: maybe we should fix at the calling site (where is GlobalWind defined?)

Reply:
We should but that needs a trunk change :)